strapi/strapi
# [ML] Pagination v4

### What does it do?

Adds pagination to the ML

### Why is it needed?

🙄 

### How to test it?

- Go to the Media library link on the left sidebar
- Add multiple assets
- Play with the navigation controls at the bottom of the page
- The list should update accordingly
